ReactWindows/js/Components/Picker/PickerIOS.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule PickerIOS
+ */
+'use strict';
```

Comment:
Can you also make sure this outputs an "UnimplementedView", i.e.:
 module.exports = require('UnimplementedView');

---
In reply to: [56820832](https://github.com/CatalystCode/react-native/pull/265#discussion_r56820832) [](ancestors = 56820832)